.github/workflows/build.yml

```diff
@@ -27,6 +27,28 @@ jobs:
           - "hooks/openfeature-hooks-opentelemetry"
           - "providers/openfeature-provider-flagd"
 
+    services:
```

Comment:
This is fine (and what we do in other contribs repos), but I'm wondering if we should try to leverage the docker-compose so we only have to bump the versions in one place. [Here's a GitHub discussion](https://github.com/orgs/community/discussions/47105#discussioncomment-4950570) about how it could be configured. I'll let you decide if you think it's worth it, but I figured I would mention it. 😄